GOVERNANCE.adoc

```diff
@@ -0,0 +1,71 @@
+This document describes the governance rules of the ServiceTalk project/organization. It is meant to be followed by all
+the repositories / sub-projects in the organization.
+
+== Contributions
+Everyone is welcome to contribute to ServiceTalk. Contributions aren't limited to submitting pull requests there are
+many different ways for you to get involved, including answering questions and reporting or triaging bugs. No matter how
+you want to get involved, we ask that you first learn what’s expected of anyone who participates in the project by
+reading the xref:CONTRIBUTING.adoc[Contributor Guidelines].
+
+== Decision Making and Voting
+ServiceTalk is an open source project, and Github serves as the source of truth for the project. Technical disagreements
+may come up, but all community members are always expected to follow the xref:CODE_OF_CONDUCT.adoc[Code of Conduct]. In
+general it is preferred that direction can be mutually agreed upon by community members. If a disagreement cannot be
+resolved independently, the <<Maintainers>> can be called in to vote on the proposal (e.g. issue/PR) in question. An
+independent link:https://github.com/apple/servicetalk/issues[github issues] should be opened prefixed with "[Vote]" and
+reference the proposal in question. The voting period should remain open for at least 2 weeks. Each Maintainer gets one
+vote and the majority will decide the direction. The final decision including the rational for the majority decision
+must be summarized by a maintainer.
+
+== Maintainers
+The ServiceTalk Maintainers are responsible for the long term maintenance of the ServiceTalk project/organization. Any
+additions to the repository are weighted on a spectrum of applicability and maintenance overhead. If a feature lies on
+the limited applicability and high maintenance end of the spectrum the maintainers tend to decide against expanding
+scope of the repository. In the future the maintainers may create a separate organization to facilitate discovery and
+collaboration for community projects. Until this time contributions can be maintained outside the ServiceTalk
+project, demonstrate maturity/adoption, and then be considered for contribution back to the ServiceTalk organization.
+
+=== Become a Maintainer
+Making contributions does not require becoming a maintainer, or obtaining commit access to the ServiceTalk
+project/organization. However if you are interested in taking a more formal role with decision making power you should
+consider becoming a Maintainer. Some characteristics of a Maintainer are as follows:
+
+* Be involved in contributing code, pull request review, triage of issues and addressing user questions in one or more
+forums such as Github, Stackoverflow, etc.
```

Comment:
nit: it's `GitHub`... Same is true for other places in the doc.

Reply:
done